HospitalToday/UI/MenuBuilder.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalToday.UI
+{
+    class MenuBuilder
+    {
+        public MenuBuilder()
+        {
+            menuBuilder = new StringBuilder();
+            menuData = new MenuData();
+        }
+
+        private StringBuilder menuBuilder;
```

Comment:
This fields can be readonly (if you are not changing their objects later)

Reply:
Changed type of access - added key word readonly